### ISSUE TITLE
(SoD)KUI nameplates cast bars does not show in pvp.

### ISSUE THREAD

**Issue by Valkyriebound** [kesava-wow/kuinameplates2#696]
Hi,

For season of discovery, KUI nameplates works absolutely fine in pve. Shows casts of mobs overhead their nameplate. However in any kind of pvp, whether duels or bgs, enemy casts do not show.

**Comment by Stimpleton** [comment]
YES, finally found someone talking about this, but didn't start happening for me until yesterday, Tuesdays patch. PvP cast bars not showing anymore but they were working perfectly before, weird that this post is from 2 weeks ago. @Valkyriebound did you find a fix?!? Everyone else seems to be working fine so leads me to believe it's KUI

**Comment by Valkyriebound** [comment]
Heya, sadly not, still not working for me. I also don't see enemy player casts on quartz.

**Comment by Valkyriebound** [comment]
No change, still not working as of 28/12.
